src/main/kotlin/com/jaoafa/vcspeaker/tools/Emoji.kt

```diff
@@ -46,6 +46,12 @@ object Emoji {
         return emojis.filter { startsWith(it.unicode) }.maxByOrNull { it.unicode.length }!!
     }
 
+    fun String.removeEmojis(): String {
+        return emojis.fold(this) { replaced, emoji ->
+            replaced.replace(emoji.unicode, "")
+        }
+    }
+
```

Comment:
```suggestion
    fun String.removeEmojis() = emojis.fold(this) { replaced, emoji ->
        replaced.replace(emoji.unicode, "")
    }

```
さっきのと同じですが、私もやってたので後で直しておきます